README.md

```
@@ -1,6 +1,5 @@
-# Ω Olympus Smart Contracts
-
-![image](https://img.shields.io/github/forks/OlympusDAO/olympus-contracts?style=social)
+# Ω Olympus Smart Contracts 
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)